schedule/migration/aarch64_regression_test_online.yaml

```diff
@@ -48,6 +48,9 @@ conditional_schedule:
         - migration/online_migration/zypper_patch
         - installation/install_service
   regression_tests:
+    REGRESSION_389ds:
```

Comment:
before this line you need add a schedule name such as test_389ds.
example:
remove_ltss:
    REGRESSION_LTSS:
      1:

And insert the schedule name test_389ds to the correct schedule position.